backend/LexBoxApi/GraphQL/LexQueries.cs

```diff
@@ -29,11 +29,11 @@ public IQueryable<Project> Projects(LexBoxDbContext context, bool withDeleted =
     {
         if (withDeleted)
         {
-            return context.Projects.IgnoreQueryFilters();
+            return context.Projects.Include(p => p.FlexProjectMetadata).IgnoreQueryFilters();
         }
         else
         {
-            return context.Projects;
+            return context.Projects.Include(p => p.FlexProjectMetadata);
```

Comment:
this isn't needed as it's handled dynamically by hot chocolate based on the request.